### ISSUE TITLE
Include Ollama binaries?

### ISSUE THREAD

**Issue by Osiris-Team** [ollama4j/ollama4j#65]
Users tend to be pretty lazy, thus including the binaries or an installation script that is automatically ran before using the API would be great!

**Comment by daguava** [comment]
The readme has a pretty good requirements section that gives exact commands for Ollama installation, I don't think it would be a good idea to begin bundling other project's binaries. If the user is too lazy to run the 1 command from the readme, maybe you don't want them as your user :) .

**Comment by Osiris-Team** [comment]
Yeah, but keep in mind that sometimes larger projects have multiple dependencies and If each of those requires user action it can get a lot quick.
